erts/emulator/beam/erl_nif.c

```diff
@@ -1083,6 +1083,17 @@ int enif_get_local_pid(ErlNifEnv* env, ERL_NIF_TERM term, ErlNifPid* pid)
     return 0;
 }
 
+void enif_set_pid_undefined(ErlNifPid* pid)
+{
+    pid->pid = am_undefined;
```

Comment:
Since it is internal only, why not use THE_NON_VALUE? Should it take an environment?